Incidents/incident-5.md

```diff
@@ -0,0 +1,165 @@
+                    "avatar": {
+                        "bodyShape": "urn:decentraland:off-chain:base-avatars:BaseMale",
+                        "wearables": [
+                            "urn:decentraland:off-chain:base-avatars:casual_hair_01",
+                            "urn:decentraland:off-chain:base-avatars:eyebrows_01",
+                            "urn:decentraland:off-chain:base-avatars:eyes_02",
+                            "urn:decentraland:off-chain:base-avatars:f_mouth_02",
+                            "urn:decentraland:off-chain:base-avatars:beard",
+                            "urn:decentraland:off-chain:base-avatars:oxford_pants",
+                            "urn:decentraland:off-chain:base-avatars:classic_shoes",
+                            "urn:decentraland:matic:collections-v2:0xae3d6cbe10cb4a0947c5e8362efb2802ab5098ab:0",
+                            "urn:decentraland:matic:collections-v2:0xc818a4a1f99ec77a35e6f51aabfc8c517acc9d95:2",
+                            "urn:decentraland:matic:collections-v2:0x5c3232279a3b85ba970f48a8408bf8b1379380fb:4"
+                        ],
+                        "snapshots": {
+                            "body": "bafkreibbcgy335g3kz5kol25brbta6bx4wbkr522yeup3k74uu2kvo2qva",
+                            "face256": "bafkreiber2htjpoqcocnvupy5mms6ww4cwjxkx6qqgsxx5ekeffwn3nbri"
+                        },
+                        "eyes": {
+                            "color": {
+                                "r": 0.37109375,
+                                "g": 0.22265625,
+                                "b": 0.1953125,
+                                "a": 1
+                            }
+                        },
+                        "hair": {
+                            "color": {
+                                "r": 0.55078125,
+                                "g": 0.125,
+                                "b": 0.078125,
+                                "a": 1
+                            }
+                        },
+                        "skin": {
+                            "color": {
+                                "r": 0.94921875,
+                                "g": 0.76171875,
+                                "b": 0.6484375,
+                                "a": 1
+                            }
+                        }
+                    },
+                    "tutorialStep": 256,
+                    "interests": [],
+                    "unclaimedName": "iceloops",
+                    "hasConnectedWeb3": true
+                }
+            ]
+        }
+    }
+]
+```
+
+
+July-15-2022 New [hotfix](https://github.com/decentraland/catalyst/releases/tag/4.8.6) version created and updated.
+
+
+## Impact 
+
+No major impact. The Catalyst **peer-ec1** was quickly rollback to `v4.8.0` and **peer-ap1** node was left out from the Load Balancer and the Realm scoring logic to avoid traffic being request/sent from/to that node. 
```

Comment:
```suggestion
No major impact. The Catalyst **peer-ec1** was quickly rolled back to `v4.8.0` and **peer-ap1** node was left out from the Load Balancer and the Realm scoring logic to avoid traffic being request/sent from/to that node. 
```